### DISCUSSION TITLE
Download configuration as archive

### DISCUSSION THREAD

**Discussion by kristofre** [Dynatrace/dynatrace-configuration-as-code#195] [Archive]
A requirement that came up from a customer was the ability to download an environment's configuration as an archive with a timestamp in the file name.

This could be handled like this:

`monaco download -e=environments.yaml -output=archive`

This would download the configuration and create a file with the name [ENVIRONMENT]_[EPOCHSTAMP].zip
The name of the environment would be as it is defined in the environments.yaml

**Comment by tatomir146** [comment]
Not the scope of monaco. Could be easily done with 3rd party tools.
